locale/en_US/countries.xml

```diff
@@ -62,15 +62,15 @@
 	<country name="Colombia" code="CO"/>
 	<country name="Comoros" code="KM"/>
 	<country name="Congo" code="CG"/>
-	<country name="Congo, The Democratic Republic of the" code="CD"/>
+	<country name="Congo (the Democratic Republic of the)" code="CD"/>
 	<country name="Cook Islands" code="CK"/>
 	<country name="Costa Rica" code="CR"/>
 	<country name="Côte d'Ivoire" code="CI"/>
 	<country name="Croatia" code="HR"/>
 	<country name="Cuba" code="CU"/>
 	<country name="Curaçao" code="CW"/>
 	<country name="Cyprus" code="CY"/>
-	<country name="Czech Republic" code="CZ"/>
+	<country name="Czechia" code="CZ"/>
```

Comment:
@mtub, yes it would be easy, but I think that if you try to compare en_US files with Debian iso-package you will see that they are not the same, because I've used here the original materials from ISO. As for Russian translation (I've checked it with Debian files) they have many differences.